app/models/member.rb

```diff
@@ -0,0 +1,6 @@
+class Member
+  attr_accessor :id
```

Comment:
i think this should be enough?
```suggestion
  attr_reader :id
```

Reply:
👍